modules/machine-translation/settings-preview-machine-translation.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class to advertize the Machine Translation module.
+ *
+ * @since 3.6
+ */
+class PLL_Settings_Preview_Machine_Translation extends PLL_Settings_Module {
+	/**
+	 * Stores the display order priority.
+	 *
+	 * @var int
+	 */
+	public $priority = 85;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 3.6
+	 *
+	 * @param PLL_Settings $polylang Polylang object.
+	 */
+	public function __construct( &$polylang ) {
+		parent::__construct(
+			$polylang,
+			array(
+				'module'        => 'machine-translation',
```

Comment:
Settings actions are based on module name.
For example https://github.com/polylang/polylang/blob/3.5.4/settings/settings.php#L260-L266
When Polylang Pro is activated the module name is the full namespaced qualified class name
https://github.com/polylang/polylang-pro/blob/ef45ad2f04073a66397fa52011d3bd78e87aa85f/modules/machine-translation/load.php#L27-L31 
This is this name which is used as key in the settings `modules` array because the class name doesn't starts with `PLL_Settings_` 
See https://github.com/polylang/polylang/blob/3.5.4/settings/settings.php#L89
It is also used as module property in the instance itself 
https://github.com/polylang/polylang/blob/3.5.4/settings/settings-module.php#L116

So sanitization make this module name unknown 
https://github.com/polylang/polylang/blob/3.5.4/settings/settings.php#L262 because `wp_syntex\\polylang_pro\\modules\\machine_translation\\module_settings` is sanitized to `wp_syntexpolylang_promodulesmachine_translationmodule_settings`

Reply:
Fixed in [ed560d7](https://github.com/polylang/polylang/pull/1420/commits/ed560d77b66a28f77553259469920d5aaeb16079).
@manooweb You were not looking at the right git branch, this system [has changed a bit](https://github.com/polylang/polylang-pro/blob/f5251bbd3e47ed4d396c8475af4c52cc045a8cf4/modules/admin-loader/admin-loader.php#L91-L92).

Reply:
@manooweb Ah! You're right, I missed it.

Reply:
Fixed in [c284291](https://github.com/polylang/polylang-pro/pull/1974/commits/c284291b6ac6e992b7b36c86aa0a807bc4ab2924).